ParabolInc/parabol
# feat(chore): Add an option to archive all done tasks

# Description

Fixes #8634 

## Demo

![2023-10-10 16 21 14](https://github.com/ParabolInc/parabol/assets/1879975/94e09e86-5f53-4b2f-a6e8-8a2fc0380951)

## Testing scenarios

- [ ] Archive all done tasks
  - Go to task dash board and see there's an option to archive tasks in the done column
  - Click it and see the modal pop up
  - Click confirm
  - Observe all tasks in the done column are archived

## Final checklist
